### DISCUSSION TITLE
The direction flow point on line to battery inverted .... Add 5.+6. PV Tracker ...and is there a way to make the display bigger ?

### DISCUSSION THREAD

**Discussion by higgy-de** [slipx06/sunsynk-power-flow-card#334] [Sunsynk/Deye]
Hello, 
thanks for the nice addon :)
I have 3 questions :

1. there a way to add more than 4 trackers ( I use 2 additional MicroInverters on my Deye`s )
2. Is there a way to make the card bigger ( i cannot read it on my little tablet without glasses :) )
3. I there a way where I can reverse the “wandering energy points” between devices?
When I charge the battery via PV, the dot moves “away from the battery” and when I pull out of the battery… 
the dot moves towards the battery…?! 
But the value is displayed with the correct sign?
-
-
![richtung](https://github.com/slipx06/sunsynk-power-flow-card/assets/158240781/6be1a7f2-4c27-49a0-b873-6e21d678a5bc)

**Comment by slipx06** [comment]
Hi. Thanks

1. Not at this stage. There is no space left on the card
2. You can enable panel_mode and then set you view type in the dashboard to panel (1card). It will use all available screen space. You can also look at the card_height and card_width options. the various options are in the readme https://slipx06.github.io/sunsynk-power-flow-card/configuration.html
3. You should enable "Invert Power" under the battery option and you can also enable "Show Absolute". The card expects a negative number for charging of the battery so you sensor needs to be inverted.

**Comment by higgy-de** [comment]
Hello slipx06,
thanks a lot for your fast help.

Solution No 2 is a little bit to high, then i have to scroll down to see battery etc :) Is there a way for something in the middle of this ? 
Solution No 3 is running good, I can live that now the Number is positiv , when battery is giving power ... (its ony the sight of view :) )

Your Answer to 1. you think there is no "graphic-space" in the card for a new "String" ?

I see no problem for that ... photoshop :) :  
![5_string](https://github.com/slipx06/sunsynk-power-flow-card/assets/158240781/31e1c953-009a-42cb-8150-446b829b999f)

**Reply by slipx06** [reply]
Try adjust the card_width and it will scale the height i.e. 

panel_mode: true
card_width: 800px

**Comment by higgy-de** [comment]
I try this ...sorry i am a HA newbie :)
Nothing change when I insert this ...
![yaml](https://github.com/slipx06/sunsynk-power-flow-card/assets/158240781/e6fc17d0-d8f7-455d-8b3e-b90939d61e87)

**Reply by slipx06** [reply]
Like this

![image](https://github.com/slipx06/sunsynk-power-flow-card/assets/7227275/23265d16-2cd1-488b-a8e5-6d19a9176226)

**Comment by higgy-de** [comment]
oh .. ok .. `?!
it looks like, it wasn safed, when i edit this...
so i have to check why it wasn in it, when i open the yaml editor again ...
Is this file somewhere in the directories of HA , then i can try to edit it there ?

**Reply by slipx06** [reply]
Those options you can set through the visual editor of the card. You dont need to edit the Dashboard yaml

![image](https://github.com/slipx06/sunsynk-power-flow-card/assets/7227275/5afea5a7-0abe-4a6c-87b2-eefab018f4e9)

**Comment by higgy-de** [comment]
OH WOW ..SORRY ...  WORKS :) Thanks a lot :) I have tomatoes in front of my eyes ? ;)
btw I try to buy you a coffee, but my bank visa thing just mailed me, that there is a problem 
( dont know my or your side ) ... do you have paypal also ?

**Reply by slipx06** [reply]
Great, Thank you. I appreciate the support. I didn't see any problems on my side 😄
